Makefile

```
@@ -1,3 +1,17 @@
+ifeq ($(OS),Windows_NT)
+SHELL := powershell.exe
+.SHELLFLAGS := -NoProfile -Command
+MKDIR := mkdir -force
+TOUCH := new-item -force
+CMAKE := cmake
+CMAKE_GENERATOR := Ninja
+else
+MKDIR := mkdir -p
+TOUCH := touch
+CMAKE := $(shell (command -v cmake3 || echo cmake))
+CMAKE_GENERATOR ?= $(shell (command -v ninja > /dev/null 2>&1 && echo "Ninja") || echo "Unix Makefiles")
+endif
```

```suggestion
ifeq ($(OS),Windows_NT)
  SHELL := powershell.exe
  .SHELLFLAGS := -NoProfile -Command
  MKDIR := mkdir -force
  TOUCH := new-item -force
  CMAKE := cmake
  CMAKE_GENERATOR := Ninja
else
  MKDIR := mkdir -p
  TOUCH := touch
  CMAKE := $(shell (command -v cmake3 || echo cmake))
  CMAKE_GENERATOR ?= $(shell (command -v ninja > /dev/null 2>&1 && echo "Ninja") || echo "Unix Makefiles")
endif
```

Note: spaces (not tabs) are used here.